common-libs/common/src/services/DidDocumentService/KeyVault.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Abstraction of a key vault used for generating did documents
+ */
+export interface KeyVault {
+  /**
+   * Primary public key
+   */
+  primaryPublicKey: Buffer
+
+  /**
+   * Recovery public key if available
+   */
+  recoveryPublicKey: Buffer
+
+  /**
+   * Array of external keys to be added to the elem object
+   */
+  externalKeys(): any[]
```

Comment:
You forgot to make this a property instead of a function :)

Otherwise everything looks fine

Reply:
done